From 6ba92ae4d916827bca36b2249af7c9eb45248789 Mon Sep 17 00:00:00 2001
From: Swarn Singh <swarnkum@codeaurora.org>
Date: Wed, 4 Aug 2021 22:44:51 +0530
Subject: [PATCH 3/3] wifi: Add vendor wifi supplicant 2.1 and 2.2 hal to
 allowed_deps

This CL adds vendor wifi supplicant 2.1 and 2.2 hal to allowed_deps to
make compiler happy.

Original change: https://keystone-qcom-review.googlesource.com/c/platform/packages/modules/Wifi/+/571427

Change-Id: Ib6499e954889125ea13620650a1a4c77f93ea3e6
CRs-Fixed: 3007678
---
 build/allowed_deps.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build/allowed_deps.txt b/build/allowed_deps.txt
index 8f07483..e724dcc 100644
--- a/build/allowed_deps.txt
+++ b/build/allowed_deps.txt
@@ -730,6 +730,8 @@ vendor.qti.hardware.wifi.hostapd-V1.0-java(minSdkVersion:current)
 vendor.qti.hardware.wifi.hostapd-V1.1-java(minSdkVersion:current)
 vendor.qti.hardware.wifi.hostapd-V1.2-java(minSdkVersion:current)
 vendor.qti.hardware.wifi.supplicant-V2.0-java(minSdkVersion:current)
+vendor.qti.hardware.wifi.supplicant-V2.1-java(minSdkVersion:current)
+vendor.qti.hardware.wifi.supplicant-V2.2-java(minSdkVersion:current)
 wifi-lite-protos(minSdkVersion:30)
 wifi-nano-protos(minSdkVersion:30)
 xz-java(minSdkVersion:current)
-- 
2.37.1

